dashboard/app/views/scripts/show.html.haml

```diff
@@ -44,14 +44,6 @@
       %br/
       %br/
 
-    - if @script.has_lesson_plan?
-      %a{href: CDO.code_org_url('/curriculum/' + @script.name)}
-        = t('home.lesson_plans')
-      %br/
-      %br/
-
-    %br/
-
```

Comment:
This removal will also affect users who aren't opted into the experiment, correct?  Will that be an issue?